backend/env/.env

```diff
@@ -0,0 +1,44 @@
+# Get your Plaid API keys from the dashboard: https://dashboard.plaid.com/team/keys
+PLAID_CLIENT_ID=6612f5a79740d6001bb774e8
+PLAID_SECRET=7d0db68493d3a138bfb2e333c9d227
```

Comment:
This file should never have been checked in, you have now exposed your client id and secret to the world, and at this point hackers would have access to your tokens to maliciously make calls to the plaid backend. If this were a real production application it would be a major security hole that would bankrupt your company.

Please take the following actions:
1. Remove this file from your repository
2. On the plaid API service, generate a new plaid client ID and/or secret, and invalidate the one you specified here.
3. Add .env to the `.gitignore` file so this never happens again.